CMakeLists.txt

```diff
@@ -107,12 +107,38 @@ if (MINGW)
 		PATHS ${win32_lib_dir}
 	)
 
-	target_link_libraries (Compare ${comctl32} ${comdlg32} ${shlwapi} ${msimg32})
+	target_link_libraries (ComparePlugin ${comctl32} ${comdlg32} ${shlwapi} ${msimg32})
 
-	install (FILES ${CMAKE_BINARY_DIR}/libCompare.dll
+	install (FILES ${CMAKE_BINARY_DIR}/libComparePlugin.dll
 		DESTINATION "$ENV{HOME}/.wine/drive_c/Program Files/Notepad++/plugins"
 		RENAME ComparePlugin.dll
 	)
+
+	if( CMAKE_SIZEOF_VOID_P EQUAL 8 )
```

Comment:
Could you please follow the current coding style:
`if (STATEMENT)` instead of
`if( STATEMENT )`?

Thank you.